src/js/apps/patients/patient/dashboard/dashboard_app.js

```diff
@@ -21,7 +19,10 @@ export default App.extend({
   },
 
   beforeStart({ patient }) {
-    const filter = { status: [STATE_STATUS.QUEUED, STATE_STATUS.STARTED].join(',') };
+    const currentOrg = Radio.request('bootstrap', 'currentOrg');
+    const states = currentOrg.getStates();
+    const filter = { state: states.filterByDone().pluck('id').join(',') };
+
```

Comment:
This doesn't work in my local sandbox.

Both the `/patients/{patientId}/relationships/actions` and `/patients/{patientId}/relationships/flows` API endpoints are still using `filter[status]` instead of `filter[state]`.

Reply:
@nmajor25 🐞 you found a bug, those endpoints are indeed still using status, and should not be. I will create a PR.